### PR TITLE
Fix Issue #403(Support to build IoT.js for Raspberry Pi on Mac OS.)

### LINKED ISSUE #403: Support to build IoT.js for Raspberry Pi on Mac OS.
Hello,

Currently, IoT.js only support to build for Raspberry Pi on Linux. 
I think it will be better if we support to build for Raspberry Pi on Mac OS, too.

For supporting build for Raspberry Pi on Mac OS,

1) Need to update document 'Build for ARM Linux (RPi2)' - for Mac OS prerequisite. 
: (https://github.com/Samsung/iotjs/wiki/Build-for-ARM-Linux-(RPi2))

2) Modify build options. - Currently, build script is failed. I found which options should be fixed, so I will make Pull request.

### PR DESCRIPTION
**Pull request by chokobole**
Fix #403

### CODE REVIEW

File: docs/Build-for-RPi2.md [hunk 1]
```diff
@@ -0,0 +1,62 @@
+## Build IoT.js with RPi2
+
+### Prerequisite
+* #### Linux
```
Comment (akosthekiss) [inline docs/Build-for-RPi2.md]: This is a very strange style (heading in a list). Just use a plain simple heading. And then the text body will need no full-tab/8-chars/4-chars indentation.

Comment (chokobole) [inline docs/Build-for-RPi2.md]: @akosthekiss  Sorry, I corrected as you mentioned.

File: docs/Build-for-RPi2.md [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+Lastly, open a shell and run with your test program.
+
+``` bash
+ssh pi@(your RPi2 IP)
+./iotjs (your test program)
+```
+
+### Building in Raspberry Pi 2 ?
+
+Not yet. Maybe someone out there?
```
Comment (LaszloLango) [inline docs/Build-for-RPi2.md]: I thought this should work after https://github.com/Samsung/libtuv/pull/28

Comment (yichoi) [inline docs/Build-for-RPi2.md]: Building in rpi2 still doesn't work - due to some architecture name issue (arm, armv7l, armv7l-hf, ...). Another PR should be created to fix this.

**Pull request merged**